blocks/footer/footer.css

```diff
@@ -1,450 +1,521 @@
+		border-bottom: 1px solid var(--primary-blue-80);
+	}
+
+	.ft-social .ft-social-list:nth-child(2) {
+		grid-column: 1 / 3;
+		grid-row: 2 / 2;
+		margin: 0;
+		padding: 0 32px 0 0;
+	}
+
+	.ft-social .ft-social-list:nth-child(2) ul {
+		justify-content: center;
+		padding: 28px 0 0;
+		width: 100%;
+	}
+
+	.ft-main .ft-main-item:last-child ul li {
+		text-align: center;
+		width: 100%;
+	}
+
+	.ft-social .ft-social-list:nth-child(2)>ul>li:not(:first-child) {
+		padding: 0 0 0 35px;
+	}
+
+	.ft-main .ft-main-item:not(:last-child) ul li:not(:last-child) {
+		padding: 0;
+	}
+
+	.ft-social .ft-social-list:last-child ul li:not(:last-child) {
+		padding-right: 12px;
+		padding-left: 0;
+	}
+
+	html[dir="rtl"] .ft-social .ft-social-list:nth-child(2)>ul>li:not(:first-child) {
+		padding: 0 35px 0 0;
+	}
+
+	.ft-social .ft-social-list:last-child {
+		grid-column: 1 / 3;
+		grid-row: 3 / 3;
+	}
+
+	.ft-social .ft-social-list:last-child ul {
+		flex-wrap: wrap;
+		justify-content: center;
+		padding: 32px 0 0;
+		border-top: 1px solid var(--primary-blue-80);
+		margin: 0;
+	}
+
+	html[dir="rtl"] .ft-social .ft-social-list:last-child ul li:not(:last-child) {
+		padding-left: 12px;
+		padding-right: 0;
+	}
+
+	html[dir="rtl"] .ft-social-list:first-child {
+		border-right: 0;
+		border-left: 0;
+	}
+
+	.ft-main .ft-main-item {
+		border-bottom: 1px solid var(--primary-blue-80);
+		padding-bottom: 2rem;
+	}
+
+	.ft-main .ft-main-item:first-child {
+		grid-column: 1 / 4;
+		grid-row: 1 / 1;
+		border: 0;
+		padding: 0;
+	}
+
+	.ft-main .ft-main-item:first-child ul {
+		justify-content: space-evenly;
+		padding: 0;
+	}
+
+	.ft-main .ft-main-item:nth-child(2) {
+		grid-column: 1 / 4;
+		grid-row: 2 / 2;
+		border: 0;
+		padding: 0;
+	}
+
+	.ft-main .ft-main-item:nth-child(2) ul {
+		justify-content: space-evenly;
+		padding: 0;
+	}
+
+	.ft-main .ft-main-item:nth-child(3) {
+		grid-column: 1 / 4;
+		grid-row: 3 / 3;
+		border: 0;
+		padding: 0;
+	}
+
+	.ft-main .ft-main-item:nth-child(3) ul {
+		justify-content: space-evenly;
+		padding: 0;
+	}
+
+	.ft-main .ft-main-item a {
+		padding: 0 0 16px;
+	}
+
+	.ft-main .ft-main-item:last-child {
+		grid-column: 1 / 4;
+		grid-row: 4 / 4;
+		border-top: 1px solid var(--primary-blue-80);
+		margin-top: 32px;
+		width: 100%;
+	}
+
+	.ft-main .ft-main-item:last-child ul {
+		flex-direction: column;
+		align-items: center;
+	}
+
+	.ft-main .ft-main-item:last-child ul li a {
+		width: 100%;
+		padding: 13px 0 16px;
+	}
+
+	.ft-main .ft-main-item:last-child p {
+		margin: 0 0 20px;
+	}
+
+
+	html[dir="rtl"] .ft-main-item:last-child p {
+		margin: 0 0 20px;
+	}
+
+	html[dir="rtl"] .ft-main-item:last-child {
+		border-left: 0;
+		border-right: 0;
+	}
+
+	.ft-legal {
+		padding: 32px 0 60px;
+	}
+
+	footer .section.bg-primary-blue-90 {
+		margin: 0;
+	}
 }
+
+@media only screen and (width > 1024px) {
+	.ft-social {
+		grid-template-columns: 1fr 1fr 2fr;
+		grid-column-gap: 0;
+	}
+
+	html[dir="rtl"] .ft-main-item:last-child {
```

Comment:
@adobe-rpal Could you please fix the indentation of this file.

Reply:
Done

Reply:
Not fixed yet. Check the diff.